ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -984,58 +1014,103 @@ func (pg *DEXOnboarding) HandleUserInteractions() {
 				return
 			}
 
-			// Initialize with password now, if dex password has not been
-			// initialized.
 			pg.isLoading = true
-			if !dexc.InitializedWithPassword() {
-				go func() {
-					// Set password.
-					pg.dexPass = []byte(pg.passwordEditor.Editor.Text())
-					if err := dexc.InitWithPassword(pg.dexPass, nil); err != nil {
+			if dexc.InitializedWithPassword() {
+				if !dexc.IsLoggedIn() {
+					pg.showDEXPasswordModal(pg.postBond)
+				} else {
```

Comment:
It shouldn't even get to the point of attempting to post a bond if the user isn't logged in.

Reply:
Updated.